### DISCUSSION TITLE
Github 약어 모음 - bnbong's tech blog

### DISCUSSION THREAD

**Discussion by giscus** [bnbong/bnbong.github.io#3] [Blog]
# Github 약어 모음 - bnbong's tech blog

Github에서 개발자들이 종종 사용하는 약어들에 대해 알아보자

https://bnbong.github.io/blog/2023/06/05/github-%EC%95%BD%EC%96%B4-%EB%AA%A8%EC%9D%8C/

**Comment by bnbong** [comment]
test comment
